Civi/Api4/Query/Api4SelectQuery.php

```diff
@@ -1014,8 +1014,7 @@ protected function autoJoinFK($key) {
     try {
       $joinPath = $joiner->getPath($explicitJoin['table'] ?? $this->getFrom(), $pathArray);
     }
-    catch (\Exception $e) {
-      // Because the select clause silently ignores unknown fields, this function shouldn't throw exceptions
```

Comment:
@eileenmcnaughton this comment should not have been deleted.

Reply:
@colemanw updated - by the way - I think we have to ignore the CI issues when upmerging conflicted commits